src/test/java/org/humancellatlas/ingest/core/service/MetadataUpdateServiceTest.java

```diff
@@ -146,6 +150,15 @@ public void testUpdateShouldNotSetStateWhenNoContent() {
         //then:
         assertThat(updatedProject).isEqualTo(project);
         verify(metadataCrudService).save(project);
-        verify(validationStateChangeService, never()).changeValidationState(project.getType(), project.getId(), ValidationState.DRAFT);
+        verify(validationStateChangeService).changeValidationState(project.getType(), project.getId(), ValidationState.DRAFT);
     }
+
+    private ObjectNode createProjectMetadataWithTitle(String title) {
```

Comment:
I believe this could be moved to a separate class for generating test objects.
It could be shared by many tests.